src/security/PeerOptions.cc

```diff
@@ -603,6 +613,28 @@ Security::PeerOptions::updateContextOptions(Security::ContextPointer &ctx) const
 #endif
 }
 
+bool
+Security::PeerOptions::updateContextCiphers(Security::ContextPointer &ctx) const
+{
+    if (sslCipher.isEmpty())
+        return true;
+
+#if USE_OPENSSL
+    const char *cipher = const_cast<SBuf*>(&sslCipher)->c_str();
+    if (!SSL_CTX_set_cipher_list(ctx.get(), cipher)) {
+        const int ssl_error = ERR_get_error();
+        debugs(83, DBG_CRITICAL, "ERROR: Failed to set SSL cipher suite '" << sslCipher << "': " <<  Security::ErrorString(ssl_error));
+        return false;
+    }
+
+    debugs(83, 5, "Using cipher suite " << sslCipher);
+
+#elif USE_GNUTLS
+    // NP: GnuTLS uses 'priorities' which are set via options= instead.
```

Comment:
s/via options=/per-session and via options=/

Reply:
done